### PR TITLE
Add install instructions to readme

### PR DESCRIPTION
**Pull request by adjohnson916**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,6 +12,26 @@ See [the demo file](demo/usage.html) for an usage example.
...
+
+### npm
+
+```
+npm install --save angular-recaptcha
+```
+
+### Manual
+
+Download `release/angular-recaptcha.js` and/or `release/angular-recaptcha.min.js`.
```
Comment (mtrias) [inline README.md]: Shouldn't this be a link to the [latest release](https://github.com/VividCortex/angular-recaptcha/releases/latest) ?

**Comment by mtrias** [comment]
Hey! thanks for your contribution.

I just made a small comment on your PR. Please let me know your thoughts.

**Comment by adjohnson916** [comment]
@mtrias I've made changes to refer to the latest release for the manual install.

**Comment by mtrias** [comment]
Thank you!

**Pull request merged**